sinonjs/samsam
# Add test for issue #71

This PR adds a test for #71 

#### How to verify

1. Check out this branch
1. `npm ci`
1. `npm run test-cloud`
1. Observe that there is no failure for "when comparing jQuery objects"
